web/js/map-OpenLayers.js

```diff
@@ -90,6 +91,34 @@ function fms_markers_list(pins, transform) {
     return markers;
 }
 
+function fms_marker_size_for_zoom(zoom) {
+    // zoomOffset seems to change between pages, so check that the zoom level
+    // we're working with is in the expected range
```

Comment:
On `/reports` zoomOffset is 0 (because you can have any zoom level) whilst on `/around` it’s 13 (the minimum zoom level), but the adding of getZoom+zoomOffset should then be the same at the same resulting zoom level on both (ie. getZoom will return 3 on `/around` and 16 on `/reports` for the same map tiles).